tests/service/views/test_utils.py

```diff
@@ -43,3 +43,30 @@ def test_result_response(svc_client):
     assert {'result'} == set(response.keys())
     assert {'datasets'} == set(response['result'].keys())
     assert ctx == response['result']
+
+
+def test_result_response_with_none(svc_client):
+    """Test result response with None value."""
+    response = result_response(DatasetListResponseRPC(), None).json
+
+    assert response
+    assert {'result'} == set(response.keys())
+    assert not response['result']
```

Comment:
Should we say `response['result'] is None`?

Reply:
Done. We should probably set this as a standard on the repository. I agree 100% with it!